Minor editorial, grammatical, and spelling improvments to docs (#1701)
vercel/turborepo: docs/pages/docs/reference/configuration.mdx

`type: boolean`

> Defaults to `true`. Whether or not to cache the task [`outputs`](#outputs-1). Setting `cache` to false is useful for daemon or long-running "watch" or development mode tasks you don't want to cache. Setting this option to `false` means that all other options will be irrelevant.

Reviewer: ```suggestion
Defaults to `true`. Whether or not to cache the task [`outputs`](#outputs-1). Setting `cache` to false is useful for daemon or long-running "watch" or development mode tasks you don't want to cache. Setting this option to `false` will ignore all other options.
```

> Defaults to `true`. Whether or not to cache the task [`outputs`](#outputs-1). Setting `cache` to false is useful for daemon or long-running "watch" or development mode tasks that you don't want to cache. Setting this option to `false` means that all other options will irrelevant.

Reviewer 2: ```suggestion
Defaults to `true`. Whether or not to cache the task [`outputs`](#outputs-1). Setting `cache` to false is useful for daemon or long-running "watch" or development mode tasks you don't want to cache. Setting this option to `false` means that all other options will be irrelevant.
```

Setting this to a list of globs will cause the task to only be rerun when files matching those globs have
changed. This can be helpful if you want to, for example, skip running tests unless a source file changed.

> Specifying `[]` will cause the task to be rerun when any version-controlled file in the workspace changes.

Reviewer 3: Same comment as above. We shouldn't change this since that is (at least aspirationally) what it should do.

Author: https://github.com/vercel/turborepo/pull/1775
